src/output.c

```diff
@@ -602,6 +603,9 @@ render_deco(struct view *view, struct output *output,
 		box = ssd_visible_box(view, LAB_SSD_BUTTON_ICONIFY);
 		render_icon(output, output_damage, &box,
 			theme->xbm_iconify_active_unpressed);
+		box = ssd_visible_box(view, LAB_SSD_BUTTON_WINDOW_MENU);
+		render_icon(output, output_damage, &box,
+			theme->xbm_maximize_active_unpressed);
```

Comment:
I'm not sure that the maximized icon looks quite right here. Aesthetically it doesn't feel quite right.
I've had a look at the openbox spec. Interestingly, they don't support a window xbm icon, so none of the themes will contain one. They just use the icon that comes with with application (and I think can be obtain via some X11 call - can't remember). Not sure if the wayland protocol supports this.
One option could be to just have an invisible icon I guess.

Reply:
> I'm not sure that the maximized icon looks quite right here. Aesthetically it doesn't feel quite right.

I agree, it was just the best match of the 3 Icons I had at my disposal :)
Personally I'd prefer just a dot in the middle.

> Interestingly, they don't support a window xbm icon, so none of the themes will contain one.
> They just use the icon that comes with with application [..] Not sure if the wayland protocol supports this.

I think (but may be wrong) it does not. I didn't really investigate any of the xdg_* specs yet. The closest to some kind of icon I came across so far is the `app_id` which is apparently being used with some informed guesses (but still guesses) to match to a `.desktop` file which then contains the name of an icon which may or may not be available in the current GTK (or Qt?) theme. Unless labwc decides to depend on GTK I'd rather not implement that 'specification'. I had to implement it within the panel but its ugly.

> One option could be to just have an invisible icon I guess.

Or maybe hardcode one like in `src/xbm/xbm.c`: `char close_button_normal[]`

Reply:
There is https://gitlab.freedesktop.org/ddevault/fdicons
so, could implement without gtk, but there is a lot said for keeping it simple and avoiding all that.

Yes, hard coding in xbc.c is a good option.

I think I like a dot too.
Alternatively, hard-code a blank icon and support reading a window.xbm or something, so anyone keen can add it to their theme

Reply:
Had a quick try at implementing this:
![window-menu-rounded](https://user-images.githubusercontent.com/35009135/150984536-1bb98bfe-1c8d-4147-8bac-7b9ee3ef2bb2.png)

I made 3 different variants which can be easily switched for testing:
- 4x4 square
- 4x4 filled square
- 4x4 filled rounded square
- Color is `window.(in)active.button.unpressed.image.color` from theme + defaults to black

Screenshot obviously uses the filled rounded square (or 'dot')
Patch should try to load a `menu.xpm` from the theme but I didn't test that.
I don't quite understand the `*_button_toggled[]` variants in `src/xbm/xbm.c`, they don't seem to be used anywhere.

Reply:
Look good. I like the dot.
The `max_button_toggled[]` is the icon to show in the maximized state - and so on.
Need to implement that one soon.